dbt/include/redshift/macros/catalog.sql

```diff
@@ -23,13 +23,28 @@
         order by "column_index"
     ),
 
+    materialized_views as (
+        select
+            table_schema  as nspname,
+            table_name    as relname
+        from information_schema.views
+        where (
+            {%- for schema in schemas -%}
+                upper(table_schema) = upper('{{ schema }}'){%- if not loop.last %} or {% endif -%}
+            {%- endfor -%}
+        )
+        and table_catalog = '{{ database }}'
+        and view_definition ilike '%create materialized view%'
```

Comment:
This will be performance intensive. Is there a faster (regex?) alternative?

Reply:
Oh, perhaps. But we're already doing this in `list_relations_without_caching`. To my knowledge, there's no good way to do this check.

Reply:
Hmmm, fair enough. I appreciate the point about maintaining state of the art for now.